docs/current/stubs.md

```diff
@@ -69,13 +69,10 @@ Creates an anonymous stub function
 
 #### `var stub = sinon.stub(object, \"method\");`
 
-Replaces `object.method` with a stub function.
+Replaces `object.method` with a stub function. An exception is thrown if the property is not already a function, to help avoid typos when stubbing methods.
```

Comment:
I agree, actually -- that clause was in there before and I didn't feel comfortable ripping it out. I'd be happy to do it though!

> On Nov 28, 2015, at 2:15 PM, Morgan Roderick <notifications@github.com> wrote:
> 
> In docs/current/stubs.md:
> 
> > @@ -69,13 +69,10 @@ Creates an anonymous stub function
> >  
> >  #### `var stub = sinon.stub(object, \"method\");`
> >  
> > -Replaces `object.method` with a stub function.
> > +Replaces `object.method` with a stub function. An exception is thrown if the property is not already a function, to help avoid typos when stubbing methods.
> I would shorten the addition to "An exception is thrown if the property is not already a function."
> 
> We generally try to steer clear of explaining motivation, and only document behaviour.
> 
> —
> Reply to this email directly or view it on GitHub.
>